app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', class_name: 'Post'
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] [REQUIRED] Kindly add foreign keys to the comments and the likes. According to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the `User` table is linked to the `Post`, `Comment`, and `Likes` with the `author_id`. Let's add that foreign key as you added for the post on line 1. Here is an example of what is expected of you:
```ruby
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id
```